client/my-sites/plugins/plugin-details-CTA/CTA-button.jsx

```diff
@@ -264,7 +283,7 @@ function onClickInstallPlugin( {
 				`/checkout/${ selectedSite.slug }/${ marketplacePlanToAdd(
 					selectedSite?.plan,
 					billingPeriod,
-					eligibleForProPlan
```

Comment:
This needs fixing everywhere https://github.com/Automattic/wp-calypso/pull/68178

Reply:
sounds good!